packages/core/package.json

```diff
@@ -97,8 +96,10 @@
     "@libp2p/interfaces": "^3.0.2",
     "@libp2p/peer-id": "^1.1.10",
     "@multiformats/multiaddr": "^11.0.6",
+    "@waku/byte-utils": "*",
     "@waku/interfaces": "*",
     "debug": "^4.3.4",
+    "fast-sha256": "^1.3.0",
```

Comment:
we already use sha3.js as a dependency. Can we use a single dependency for sha functionality?

Best to use the same dependency that the other crypto deps we have (e.g. @noble/*)